web/themes/custom/fmcg_material/sass/_checkout.scss

```diff
@@ -0,0 +1,220 @@
+.layout-checkout-form {
+  background: $fmcg-black;
+  box-shadow: none;
+
+  .checkout-pane {
+    padding-right: 24px;
+  }
+
+  a {
+    color: $fmcg-chrome;
+    font-size: 14px;
+    vertical-align: middle;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $fmcg-yellow;
+    }
+  }
+
+  .form-item input[type=text] {
+    border: 1px solid $fmcg-yellow;
+  }
+
+  div.form-item label {
+    font-size: 14px;
+    line-height: 2em;
+    color: $fmcg-chrome;
+  }
+
+  .form-item.form-type-select:after {
+    content: "arrow_drop_down";
+    margin-left: -24px;
+    margin-top: 8px;
+    color: rgba(0, 0, 0, 0.54);
+    z-index: 1;
+    position: absolute;
+    right: 0;
+    top: 26px;
+  }
+
+  span.fieldset-legend {
+    background-color: $fmcg-black;
+
+    &::after {
+      content: "";
+      position: absolute;
+      height: 1px;
+      background-color: #FFFFFF;
+      width: 100%;
+      top: 50%;
+      left: 0;
+      right: 0;
+      z-index: -1;
+    }
+  }
+
+  legend {
+    position: relative;
+    width: 100%;
+    z-index: 5;
+    background: none;
+    margin-bottom: 12px;
+  }
+
+  label.option {
+    color: #FFFFFF;
+  }
+
+  .fieldset-wrapper {
+    color: $fmcg-chrome;
+    font-family: Roboto;
+    font-size: 13px;
+  }
+
+  div.form-item > label {
+    width: 100%;
+  }
+
+  fieldset.payment-method legend {
+    display: none;
+  }
+
+  .checkout-address-book-edit-button {
+    width: min-content !important;
+    height: min-content;
+  }
+
+  .form-item-payment-information-payment-method {
+    display: inline-block;
+    margin-right: 12px;
+    padding: 4px 12px;
+    background: #FF944D;
+    border-radius: 2px;
+  }
+
+  .fieldset-legend {
+    color: $fmcg-yellow;
+    font-family: Poppins;
+    font-weight: 600;
+    font-size: 22px;
+  }
+
+  .form-item input[type=radio]:checked:before {
+    content: "radio_button_checked";
+    color: #2A556B;
+  }
+  .form-item input[type=radio]:before {
+    content: "radio_button_unchecked";
+    padding-top: 5px;
+  }
+}
+
+.layout-region {
```

Comment:
Is this true for all layouts everywhere, or just when they're inside the checkout?

Reply:
This is specifically for the checkout, so I've now wrapped this in the appropriate class.